src/components/App.js

```diff
@@ -10,9 +10,11 @@ import MovieDetails from './MovieDetails'
 function App() {
   return (
     <Router>
-      <Route path="/" component={HeaderNav} />
+      <Switch>
```

Comment:
I'm not sure what you are trying to accomplish with this routing scheme.

From how I'm reading it, the MovieDetails screen won't have a header. But you will also render the Home component below the Movie Details?